tests/unit/test_medication_reminder_schema.py

```
@@ -0,0 +1,103 @@
+        assert "Aspirin" in title
+        assert "(" not in body  # no empty dosage parens
+
+    def test_is_test_flag_produces_test_messaging(self):
+        title, body = medication_reminder_due_template(
+            {"medication_name": "Aspirin", "is_test": True}
+        )
+        assert "Test" in title
+        assert "test" in body.lower()
+
+    def test_missing_fields_fall_back_to_placeholders(self):
+        title, body = medication_reminder_due_template({})
```

Ruff (0.15.15)

[warning] Unpacked variable `title` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)